frontend/src/diagram/DiagramWebSocketContainer.tsx

```diff
@@ -229,15 +231,56 @@ export const DiagramWebSocketContainer = ({ representationId, selection, setSele
     zoomLevel,
     subscribers,
     message,
+    errorMessage,
   } = state;
 
-  const [deleteElementsMutation] = useMutation(deleteFromDiagramMutation);
-  const [invokeNodeToolMutation] = useMutation(invokeNodeToolOnDiagramMutation);
-  const [invokeEdgeToolMutation] = useMutation(invokeEdgeToolOnDiagramMutation);
-  const [editLabelMutation] = useMutation(editLabelMutationOp);
+  const [deleteElementsMutation, deleteElementResult] = useMutation(deleteFromDiagramMutation);
+  const [invokeNodeToolMutation, invokeNodeToolResult] = useMutation(invokeNodeToolOnDiagramMutation);
+  const [invokeEdgeToolMutation, invokeEdgeToolResult] = useMutation(invokeEdgeToolOnDiagramMutation);
+  const [editLabelMutation, editLabelResult] = useMutation(editLabelMutationOp);
   const [getToolSectionData, { loading: toolSectionLoading, data: toolSectionData }] = useLazyQuery(
     getToolSectionsQuery
   );
+
+  const setErrorMessage = useCallback((errorMessage?) => {
+    dispatch({ type: HANDLE_ERROR_MESSAGE__ACTION, errorMessage });
+  }, []);
+
+  useEffect(() => {
+    if (!deleteElementResult?.loading && deleteElementResult?.data?.data) {
```

Comment:
I beliave the double `data` (here and below) is wrong now that we use Apollo Client.